web-src/src/components/MainSidePanel.js

```diff
@@ -205,14 +205,20 @@ export function MainSidePanel(props) {
               && sessions.toReversed().map((session) => {
                 const sessionDate = new Date(session.name);
                 const groupingLabel = groupingLabelGenerator(sessionDate);
-
-                return (<Item key={sessionDate}>
-                  {groupingLabel && <Text UNSAFE_className={style.subMenuHeader}>{groupingLabel}</Text>}
-                  {/* eslint-disable-next-line max-len */}
-                  <li className={currentSession && viewType === ViewType.CurrentSession && session && session.id === currentSession.id ? derivedStyles.clickedSubMenuItem : style.subMenuItem} key={session.id}>
-                    <Link href="#" UNSAFE_className={style.menuItemLink} onPress={() => handleRecent(session)}>{session.name}</Link>
-                  </li>
-                </Item>);
+                return (
+                      <>
+                        {groupingLabel
+                            && <Text key={groupingLabel} UNSAFE_className={style.subMenuHeader}>
+                              {groupingLabel}
+                            </Text>}
+                        {/* eslint-disable-next-line max-len */}
+                        <li className={currentSession && viewType === ViewType.CurrentSession && session && session.id === currentSession.id ? derivedStyles.clickedSubMenuItem : style.subMenuItem}
+                            key={session.id}>
+                          <Link href="#" UNSAFE_className={style.menuItemLink}
+                                onPress={() => handleRecent(session)}>{session.name}</Link>
+                        </li>
+                      </>
```

Comment:
Noticed some extra spacing here. Looks like a task for code formatter.

Reply:
I'll check if we can update linter to fix this automatically